### ISSUE TITLE
Input format

### ISSUE THREAD

**Issue by gkhom** [DANS-KNAW/easy-ingest#3]
Implement input format as a "digital set folder" that contains multiple folders where each represents a digital object (DO). Each DO folder contains a `fo.xml` file, multiple files that are datastreams on the DO and a config file (do.config?) which has the following format:

``` properties
easy-dataset                        -- the namespace of the DO
fedora:isMemberOf DO_FOLDER1        -- relation (predicate, object)
fedora:isSubordinateTo DO_FOLDER2   -- another relation
```

**Comment by gkhom** [comment]
This is done in #5 
Small change: I named the config `dc.cfg`

@janvanmansum if the format and implementation is ok, we can close this.

**Comment by gkhom** [comment]
We decided to switch to a JSON format because we need to supply mime-types as well, and possibly more things in the future. An example of the new format:

``` Javascript
{
  "namespace" : "easy-dataset",
  "files" : [
    {
      "filename" : "DATASET_LICENSE",
      "mime" : "application/pdf"
    },
    {
      "filename" : "AMD",
      "mime" : "application/xml"
    },
    {
      "filename" : "quicksort.hs",
      "mime" : "text/plain"
    }
  ],
  "relations" : [
    {
      "predicate": "fedora:isMemberOf",
      "objectName" : "do1"
    },
    {
      "predicate": "fedora:isSubordinateTo",
      "objectName" : "do1"
    }
  ]
}
```

This is implemented in #9

**Comment by janvanmansum** [comment]
@GeorgiKhomeriki 

I have further tweaked and specified your json-format. See the new specs. Please review them. I will try to create issues for all the non-trivial items of work I can think of.
